exercises/practice/say/.meta/proof.ci.js

```diff
@@ -1,90 +1,82 @@
-export class Say {
-  constructor() {
-    this.smallNumbers = {
-      0: 'zero',
-      1: 'one',
-      2: 'two',
-      3: 'three',
-      4: 'four',
-      5: 'five',
-      6: 'six',
-      7: 'seven',
-      8: 'eight',
-      9: 'nine',
-      10: 'ten',
-      11: 'eleven',
-      12: 'twelve',
-      13: 'thirteen',
-      14: 'fourteen',
-      15: 'fifteen',
-      16: 'sixteen',
-      17: 'seventeen',
-      18: 'eighteen',
-      19: 'nineteen',
-    };
+export const say = (n) => {
+  let result;
+  const number = { current: n };
+  let smallNumbers = {
```

Comment:
All the fixed objects and helper methods should be defined outside of the `say` function as `const`.